tests/AerospikeCacheTest.php

```diff
@@ -3,7 +3,200 @@
 
 namespace Lmc\AerospikeCache;
 
-class AerospikeCacheTest
+use PHPUnit\Framework\TestCase;
+
+class AerospikeCacheTest extends TestCase
 {
+    /**
+     * @dataProvider provideStatusCodes
+     */
+    public function testShouldConfirmItemPresence(int $statusCode, bool $expectedReturnedValue): void
+    {
+        $aerospikeMock = $this->createMock(\Aerospike::class);
```

Comment:
This 2 lines are the same in every test, it could/should be move to `setUp` method.

Reply:
moved to the setup

Reply:
Moved it to setup